docs/Manual/Deployment/Kubernetes/Drain.md

```diff
@@ -0,0 +1,444 @@
+{% endhint %}
+
+One possibility to verify these two things is via the ArangoDB web interface.
+Node health can be monitored in the _Overview_ tab under _NODES_:
+
+![Cluster Health Screen](./HealthyCluster.png)
+
+**Check that all nodes are green** and that there is **no node error** in the
+top right corner.
+
+As to the shards being in sync, see the _Shards_ tab under _NODES_:
+
+![Shard Screen](./ShardsInSync.png)
+
+**Check that all collections have a green check mark** on the right side.
+If any collection does not have such a check mark, you can click on the
+collection and see the details about shards. Please keep in
+mind that this has to be done **for each database** separately!
+
+Obviously, this might be tedious and calls for automation. Therefore, there
+are APIs for this. The first one is [Cluster Health](../../../HTTP/Cluster/Health.html):
+
+```
+POST /_admin/cluster/health
+```
+
+… which returns a JSON document looking like this:
+
+```JSON
+{
+  "Health": {
+    "CRDN-rxtu5pku": {
+      "Endpoint": "ssl://my-arangodb-cluster-coordinator-rxtu5pku.my-arangodb-cluster-int.default.svc:8529",
+      "LastAckedTime": "2019-02-20T08:09:22Z",
+      "SyncTime": "2019-02-20T08:09:21Z",
+      "Version": "3.4.2-1",
+      "Engine": "rocksdb",
+      "ShortName": "Coordinator0002",
+      "Timestamp": "2019-02-20T08:09:22Z",
+      "Status": "GOOD",
+      "SyncStatus": "SERVING",
+      "Host": "my-arangodb-cluster-coordinator-rxtu5pku.my-arangodb-cluster-int.default.svc",
+      "Role": "Coordinator",
+      "CanBeDeleted": false
+    },
+    "PRMR-wbsq47rz": {
+      "LastAckedTime": "2019-02-21T09:14:24Z",
+      "Endpoint": "ssl://my-arangodb-cluster-dbserver-wbsq47rz.my-arangodb-cluster-int.default.svc:8529",
+      "SyncTime": "2019-02-21T09:14:24Z",
+      "Version": "3.4.2-1",
+      "Host": "my-arangodb-cluster-dbserver-wbsq47rz.my-arangodb-cluster-int.default.svc",
+      "Timestamp": "2019-02-21T09:14:24Z",
+      "Status": "GOOD",
+      "SyncStatus": "SERVING",
+      "Engine": "rocksdb",
+      "ShortName": "DBServer0006",
+      "Role": "DBServer",
+      "CanBeDeleted": false
+    },
+    "AGNT-wrqmwpuw": {
+      "Endpoint": "ssl://my-arangodb-cluster-agent-wrqmwpuw.my-arangodb-cluster-int.default.svc:8529",
+      "Role": "Agent",
+      "CanBeDeleted": false,
+      "Version": "3.4.2-1",
+      "Engine": "rocksdb",
+      "Leader": "AGNT-oqohp3od",
+      "Status": "GOOD",
+      "LastAckedTime": 0.312
+    },
+    ... [some more entries, one for each instance]
+  },
+  "ClusterId": "210a0536-fd28-46de-b77f-e8882d6d7078",
+  "error": false,
+  "code": 200
+}
+```
+
+Check that each instance has a `Status` field with the value `"GOOD"`.
+Here is a shell command which makes this check easy, using the
+[`jq` JSON pretty printer](https://stedolan.github.io/jq/):
+
+```bash
+curl -k https://arangodb.9hoeffer.de:8529/_admin/cluster/health --user root: | jq . | grep '"Status"' | grep -v '"GOOD"'
```

Comment:
@neunhoef Does jq support something similar to grep, so that we can make this a cross-platform command?